code/modules/clothing/spacesuits/rig/suits/ert.dm

```diff
@@ -98,7 +98,7 @@
 
 /obj/item/rig/ert/assetprotection
 	name = "heavy asset protection suit control module"
-	desc = "A heavy suit worn by the highest level of Asset Protection, don't mess with the person wearing this. Armoured and space ready."
+	desc = "A heavy suit worn by the highest level of Asset Protection, don't mess with the person wearing this. armored and space ready."
```

Comment:
```suggestion
	desc = "A heavy suit worn by the highest level of Asset Protection, don't mess with the person wearing this. Armored and space ready."
```

should probably fix the other armors that start with caps at the front

Reply:
Done, thanks.